### PR TITLE
Hotfix 4.1.11

### PR DESCRIPTION
**Pull request by ZeroStride**
* Log out ISO8601 date time during sdk_init

### CODE REVIEW

File: Teak/TeakLog.m [hunk 1]
```diff
@@ -113,7 +113,16 @@ - (nullable id)initForTeak:(nonnull Teak*)teak withAppId:(nonnull NSString*)appI
...
-  [self logEvent:@"sdk_init" level:INFO eventData:nil];
+
+  // Log ISO8601 format timestamp at init
+  NSDateFormatter* formatter = [[NSDateFormatter alloc] init];
+  [formatter setLocale:[NSLocale localeWithLocaleIdentifier:@"en_US_POSIX"]];
+  formatter.timeZone = [NSTimeZone timeZoneWithAbbreviation:@"UTC"];
+  [formatter setDateFormat:@"yyyy-MM-dd'T'HH:mm:ss"];
+  NSString* iso8601String = [formatter stringFromDate:[NSDate date]];
+  
+  NSDictionary* dateTime = @{@"at" : [iso8601String stringByAppendingString:@"Z"]};
```
Comment (AlexSc) [inline Teak/TeakLog.m:124]: Why not just put the 'Z' in as part of the date format?

File: Teak/TeakLog.m [hunk 2]
```diff
@@ -113,7 +113,16 @@ - (nullable id)initForTeak:(nonnull Teak*)teak withAppId:(nonnull NSString*)appI
...
+  [self logEvent:@"sdk_init" level:INFO eventData:@{@"at" : dateTime}];
```
Comment (AlexSc) [inline Teak/TeakLog.m:125]: dateTime is already a dictionary, so this will give us `at.at`.

**Review by AlexSc — CHANGES_REQUESTED** [review]

File: Teak/TeakLog.m (hunk 1 above)
Comment (ZeroStride) [reply Teak/TeakLog.m:124]: It doesn't format that properly. It will be something like @"2022-03-08T01:49:57+0000" instead of @"2022-03-08T01:48:31Z"

Comment (AlexSc) [reply Teak/TeakLog.m:124]: The Z should be in '' like the T, so 'Z'

Comment (ZeroStride) [reply Teak/TeakLog.m]: Oh duh. Right.

File: Teak/TeakSession.m [hunk 3]
```diff
@@ -277,7 +277,7 @@ - (void)sendUserIdentifier {
                                                       NSString* deepLink = reply[@"deep_link"];
                                                       NSURL* url = [NSURL URLWithString:deepLink];
                                                       if (url && blockSelf.launchDataOperation != nil) {
-                                                        [blockSelf.launchDataOperation.result updateDeepLink:url];
+                                                        blockSelf.launchDataOperation = [blockSelf.launchDataOperation updateDeepLink:url];
```
Comment (AlexSc) [inline Teak/TeakSession.m:280]: This will suppress the original short URL, whereas we preserve it on Android.

**Review by AlexSc — CHANGES_REQUESTED** [review]

File: Teak/TeakSession.m (hunk 3 above)
Comment (ZeroStride) [reply Teak/TeakSession.m:280]: Yep. There were some bigger changes needed to preserve it, and so I decided I'd do it in a future update.

Comment (ZeroStride) [reply Teak/TeakSession.m]: Changes made.

**Pull request merged**